assets/section-main-product.css

```diff
@@ -1068,7 +1068,7 @@ a.product__text {
 
 .thumbnail[aria-current]:focus:not(:focus-visible) {
   outline: 0;
-  box-shadow: none;
+  box-shadow: 0 0 0 0.1rem rgb(var(--color-foreground));
```

Comment:
If the `box-shadow` on `.thumbnail[aria-current]` (line 1054) is also `0.1rem`, does this line actually makes any (visible) difference? Or is the idea here just to overwrite the `box-shadow` value set by the selector above (`.thumbnail[aria-current]:focus`)?

I'm asking because you mentioned:
> when a user uses the mouse to select the thumbnail, **the border should increase in size**

in the PR description, but it looks like it doesn't increase in size when focused. It's `0.1rem` with or without mouse focus.

Reply:
> For active items, it's 0.1rem with or without mouse focus

Yes exactly. 

If `:focus-visible` only: Add outer focus ring box shadow 
If `[aria-current]` only: Add `0.1rem` box shadow 
If `[aria-current]:focus`: Add `0.1rem` box shadow. This is a mouse click, and we don't want to show focus rings.
If `[aria-current]:focus-visible`: Default to `focus-visible` styling. Once focus is removed, re-add the `[aria-current]` box-shadow.